File: find-length-of-number/javascript-typescript/using-loop.ts
function findLengthUsingDivisionLoop() {
    let num = 123456;
    let counter = 0;

    while (num > 0) {
        num = (num / 10) | 0;       // Bitwise operator to convert decimal number (float) to whole number (int)
        counter += 1;
    }

    console.log(`Final Length: ${counter}`)
}

findLengthUsingDivisionLoop();

File: find-length-of-number/c#/using-loop.cs
using System;

public class NumberLength
{
    public static void Main(string[] args)
    {
        int num = 123456;
        int counter = 0;

        while(num > 0)
        {
            num /= 10;
            counter++;
        }
        
        Console.WriteLine("Length of number is " + counter);
    }
}